charts/pega/config/deploy/server.xml.tmpl

```diff
@@ -77,6 +77,21 @@
     <Connector port="8081" protocol="org.apache.coyote.http11.Http11Nio2Protocol"
                connectionTimeout="20000"
 	       maxThreads="1"/>
```

Comment:
Renaming this file should not cause any issue inside container while dockerizing. Please verify this.

Reply:
Will test this scenario